### PR TITLE
feat(rs-dpp): migrate fees from js-dpp v0.24

### PR DESCRIPTION
**Pull request by qrayven**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-dpp/src/state_transition/validation/validate_state_transition_fee.rs [hunk 1]
```diff
@@ -85,36 +93,39 @@ where
                 if execution_context.is_dry_run() {
                     return Ok(result);
                 }
-                balance + identity.get_balance()
+
+                // TODO fixme: remove i128. what if identity_balance is negative and absolute value is
+                // TODO greater than  balance (which is u64)
+                balance as i128 + identity_balance as i128
```
[the remark this answers is not in the document]
Comment (shumkov) [reply packages/rs-dpp/src/state_transition/validation/validate_state_transition_fee.rs:99]: I think you shouldn't use i128, but you can check if `identity_balance` is negative and bigger than `balance` then you should return BalanceIsNotEnoughError right from here.

[the remark this answers is not in the document]
Comment (qrayven) [reply packages/rs-dpp/src/state_transition/validation/validate_state_transition_fee.rs:99]: Great, thank you!, Fixed

[the remark this answers is not in the document]
Comment (qrayven) [reply packages/rs-dpp/src/state_transition/validation/validate_state_transition_fee.rs]: Great, thank you!, Fixed

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**